### ISSUE TITLE
Sorting changes column width

### ISSUE THREAD

**Issue by synaptikDev** [ssuperczynski/ngx-easy-table#127]
When clicking the column header for sorting, the column expands due to the fact an arrow is added after the column title text.  This causes the table columns to 'jump' in size.

Is there a way to control this without having to edit the directive HTML directly?  Based on the current CSS, I don't see a way to override this.

**Comment by ssuperczynski** [comment]
Indeed it needs to be fixed.
As a workaround there are two options for now (if you use v4.0)
- in the config set flag `fixedColumnWidth: true,`
- in the component.ts you can add fixed width of each column:

```ts
  columns = [
    { key: 'phone', title: 'Phone', placeholder: 'Search', width: '15%' },
    { key: 'age', title: 'Age', placeholder: 'Søg', width: '10%' },
    { key: 'company', title: 'Company', placeholder: 'Pesquisa', width: '15%' },
    { key: 'name', title: 'Name', placeholder: 'поиск', width: '15%' },
    { key: 'isActive', title: 'STATUS', placeholder: 'Suche', width: '15%' },
  ];
```

**Comment by ssuperczynski** [comment]
Fixed in 5.0
